examples/center_of_mass_sparse.ipynb

```diff
@@ -0,0 +1,275 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget\n",
+    "\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import imageio\n",
+    "\n",
+    "import stempy.io as stio\n",
+    "import stempy.image as stim"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def phase_from_com(com, theta=0, flip=False, reg=1e-10):\n",
+    "    \"\"\"Integrate 4D-STEM centre of mass (DPC) measurements to calculate\n",
+    "    object phase. Assumes a three dimensional array com, with the final\n",
+    "    two dimensions corresponding to the image and the first dimension \n",
+    "    of the array corresponding to the y and x centre of mass respectively.\n",
+    "    Note this version of the reconstruction is not quantitative.\n",
+    "       \n",
+    "    author: Hamish Brown\n",
+    "    \n",
+    "    Parameters\n",
+    "    ----------\n",
+    "        com : ndarray, 3D\n",
+    "            The center of mass for each frame as a 3D array of size [2, M, N]\n",
+    "        theta : float\n",
+    "            The angle between real space and reciprocal space in radians\n",
+    "        flip : bool\n",
+    "            Whether to flip the com direction to account for a mirror across the vertical axis.\n",
+    "        reg : float\n",
+    "            A regularization parameter\n",
+    "            \n",
+    "    Returns\n",
+    "    -------\n",
+    "        : ndarray, 2D\n",
+    "            A 2D ndarray of the DPC phase.\n",
+    "    \n",
+    "    \"\"\"\n",
+    "    # Perform rotation and flipping if needed (from py4dstem)\n",
+    "    CoMx = com[0,]\n",
+    "    CoMy = com[1,]\n",
+    "    if not flip:\n",
+    "        CoMx_rot = CoMx*np.cos(theta) - CoMy*np.sin(theta)\n",
+    "        CoMy_rot = CoMx*np.sin(theta) + CoMy*np.cos(theta)\n",
+    "    if flip:\n",
+    "        CoMx_rot = CoMx*np.cos(theta) + CoMy*np.sin(theta)\n",
+    "        CoMy_rot = CoMx*np.sin(theta) - CoMy*np.cos(theta)\n",
+    "    \n",
+    "    # Get shape of arrays\n",
+    "    ny, nx = com.shape[1:]\n",
+    "\n",
+    "    # Calculate Fourier coordinates for array\n",
+    "    ky, kx = [np.fft.fftfreq(x) for x in [ny,nx]]\n",
+    "\n",
+    "    # Calculate numerator and denominator expressions for solution of \n",
+    "    # phase from centre of mass measurements\n",
+    "    numerator = ky[:,None]*np.fft.fft2(CoMx_rot)+kx[None,:]*np.fft.fft2(CoMy_rot)\n",
+    "    denominator = 2*np.pi*1j*((kx**2)[None,:]+(ky**2)[:,None])+reg\n",
+    "    # Avoid a divide by zero for the origin of the Fourier coordinates\n",
+    "    numerator[0,0] = 0\n",
+    "    denominator[0,0] = 1\n",
+    "\n",
+    "    # Return real part of the inverse Fourier transform\n",
+    "    return np.real(np.fft.ifft2(numerator/denominator))\n"
```

Comment:
I guess this can be replace with a call the stempy, now that you added it?

Reply:
yes. That was the plan.  I forgot to mention that. Thanks!

Just please give credit to [py4DSTEM](https://github.com/py4dstem/py4DSTEM/blob/72f95b2c4a5b1d62ee9e491df88a7c3ba7487261/py4DSTEM/process/dpc/dpc.py#L219) and Hamish Brown.

Reply:
This is added #199

Reply:
OK, we can go this in a follow up PR.

Reply:
Once the phase_from_com is in stempy I can test it and remove this.